Code/Framework/AzFramework/Platform/Linux/platform_linux.cmake

```diff
@@ -5,3 +5,30 @@
 # SPDX-License-Identifier: Apache-2.0 OR MIT
 #
 #
+
+# Based on the linux window manager trait, perform the appropriate additional build configurations
+# Only 'xcb', 'wayland', and 'xlib' are recognized
+if (${PAL_TRAIT_LINUX_WINDOW_MANAGER} STREQUAL "xcb")
+
+    find_library(XCB_LIBRARY xcb)
+
+    set(LY_BUILD_DEPENDENCIES
+        PRIVATE
+            ${XCB_LIBRARY}
+    )
+
+    set(LY_COMPILE_DEFINITIONS PUBLIC AZ_LINUX_WINDOW_MANAGER_XCB)
+
+elseif(PAL_TRAIT_LINUX_WINDOW_MANAGER STREQUAL "wayland")
+
+    set(LY_COMPILE_DEFINITIONS PUBLIC AZ_LINUX_WINDOW_MANAGER_WAYLAND)
+
+elseif(PAL_TRAIT_LINUX_WINDOW_MANAGER STREQUAL "xlib")
+
+    set(LY_COMPILE_DEFINITIONS PUBLIC AZ_LINUX_WINDOW_MANAGER_XLIB)
+
+else()
+
+    message(FATAL_ERROR, "Linux Window Manager ${PAL_TRAIT_LINUX_WINDOW_MANAGER} is not recognized")
+
+endif()
```

Comment:
nitpick: Can you add a newline to this file?

Reply:
Done